### PR TITLE
add plot_all_traces

### PR DESCRIPTION
**Pull request by davidgodinez**

### CODE REVIEW

File: pipeline/mtl_analysis/helper_functions.py [hunk 1]
```diff
@@ -70,6 +70,144 @@ def min_dist(x1, x2):
...
+        print(f'Mismatch in tracking trial and ephys trial number: {session_key}')
+        return
+    
+    session_traces_w = (oralfacial_analysis.WhiskerSVD & session_key).fetch('mot_svd')
+    
+    if len(session_traces_w[0][:,0]) % num_frame != 0:
+        print('Bad videos in bottom view')
+        return
+    else:
+        num_trial_w = int(len(session_traces_w[0][:,0])/num_frame)
```
Comment (dimitri-yatsenko) [inline pipeline/mtl_analysis/helper_functions.py:119]:
```suggestion
        num_trial_w = len(session_traces_w[0][:,0]) // num_frame
```

File: pipeline/mtl_analysis/helper_functions.py [hunk 2]
```diff
@@ -70,6 +70,144 @@ def min_dist(x1, x2):
...
+        session_traces_w = np.reshape(session_traces_w[0][:,0], (num_trial_w, num_frame))
+        trial_idx_nat = [d.astype(str) for d in np.arange(num_trial_w)]
+        trial_idx_nat = sorted(range(len(trial_idx_nat)), key=lambda k: trial_idx_nat[k])
+        trial_idx_nat = sorted(range(len(trial_idx_nat)), key=lambda k: trial_idx_nat[k])
+        session_traces_w=session_traces_w[trial_idx_nat,:]
+    
+    tracking_fs = float((tracking.TrackingDevice & tracking.Tracking & camera_key & session_key).fetch1('sampling_rate'))
+
+    trial_trk_1 = trk & 'water_port="mtl-1"'
```
Comment (dimitri-yatsenko) [inline pipeline/mtl_analysis/helper_functions.py:128]: There is gotta be a more parametric way of doing this enumeration.

File: pipeline/mtl_analysis/helper_functions.py [hunk 3]
```diff
@@ -70,6 +70,144 @@ def min_dist(x1, x2):
...
+    
+    session_traces_w = (oralfacial_analysis.WhiskerSVD & session_key).fetch('mot_svd')
+    
+    if len(session_traces_w[0][:,0]) % num_frame != 0:
+        print('Bad videos in bottom view')
+        return
+    else:
+        num_trial_w = int(len(session_traces_w[0][:,0])/num_frame)
+        session_traces_w = np.reshape(session_traces_w[0][:,0], (num_trial_w, num_frame))
+        trial_idx_nat = [d.astype(str) for d in np.arange(num_trial_w)]
```
Comment (dimitri-yatsenko) [inline pipeline/mtl_analysis/helper_functions.py:121]:
```suggestion
        trial_idx_nat = [str(i) for in range(num_trial_w)]
```

File: pipeline/mtl_analysis/helper_functions.py [hunk 4]
```diff
@@ -70,6 +70,144 @@ def min_dist(x1, x2):
...
+        trial_idx_nat = sorted(range(len(trial_idx_nat)), key=lambda k: trial_idx_nat[k])
```
Comment (dimitri-yatsenko) [inline pipeline/mtl_analysis/helper_functions.py:122]: `trial_idx_nat` is already sorted. What does this do?

File: pipeline/reference/__init__ 2.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+from .ccf_ontology import ccf_ontology
+
+__all__ = [ccf_ontology]
```
Comment (ttngu207) [inline pipeline/reference/__init__ 2.py:4]: Where is this file coming from? I don't think it's needed here (if not, please delete)

Comment (davidgodinez) [reply pipeline/reference/__init__ 2.py:4]: Will do.

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**